src/components/UserProfile/ProfileHeader/index.tsx

```diff
@@ -65,9 +65,8 @@ const ProfileHeader: React.FC<ProfileHeaderProps> = ({
                 {user.displayName}
               </a>
             </Link>
-            {user.displayName.toLowerCase() !== user.email.toLowerCase() &&
-              (user.id === loggedInUser?.id ||
-                hasPermission(Permission.MANAGE_USERS)) && (
+            {user.email &&
+              user.displayName.toLowerCase() !== user.email.toLowerCase() && (
```

Comment:
Why not just force the response from the api endpoint to lowercase everything so we don't need to do this all over the app?

Reply:
@sct Ahh, good call!